### PR TITLE
Add HDD User Telemetry HOL and images

### PR DESCRIPTION
**Pull request by sawpresto**
Let me know if you have questions!

### CODE REVIEW

File: docs/HOL_PartsUnlimited_HDD-User-Telemetry/HOL_PartsUnlimited_HDD-User-Telemetry.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+- Application Insights created in the same Azure Resource Group as the PartsUnlimited website
+
+- Continuous Integration build with deployment to the Azure web app
+
+**Tasks Overview**
+
+1. Set up Application Insights for PartsUnlimited
+
+2. View real-time results for user telemtry in the Azure portal
```
Comment (dtzar) [inline docs/HOL_PartsUnlimited_HDD-User-Telemetry/HOL_PartsUnlimited_HDD-User-Telemetry.md]: Typo

File: docs/HOL_PartsUnlimited_HDD-User-Telemetry/HOL_PartsUnlimited_HDD-User-Telemetry.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+**Step 12.** In the "Changes" tile in Team Explorer, commit the changes of adding Application Insights (adding ApplicationInsights.config, editing the publish profile, _Layout.cshtml, packages.config and web.config) and sync to the repo which should kick off an automated build and deploy to the website. 
+
+![](<media/prereq-step12a.png>)
+
+![](<media/prereq-step12b.png>)
+
+Now that the telemetry has been added to the web application, it may take a few minutes for Application Insights to refresh.
+
+###Task 2: View real-time results for user telemtry in the Azure portal
```
Comment (dtzar) [inline docs/HOL_PartsUnlimited_HDD-User-Telemetry/HOL_PartsUnlimited_HDD-User-Telemetry.md]: Typo

File: docs/HOL_PartsUnlimited_HDD-User-Telemetry/HOL_PartsUnlimited_HDD-User-Telemetry.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+**Step 11.** The Page views panel will break down the total page views, number of users, and pages per session for the web application.
+
+![](<media/task2step11.png>)
+
+All of the out-of-box data from Application Insights provide different facets of user telemetry to find out how users interact with your web application. 
+
+In this lab, you learned about setting up Application Insights and inserting the snippet of code for usage tracking into a web application to gain further insight into how users are behaving towards your web application.
+
+Try these labs out for next steps:
+
```
Comment (dtzar) [inline docs/HOL_PartsUnlimited_HDD-User-Telemetry/HOL_PartsUnlimited_HDD-User-Telemetry.md]: Either link to labs which actually exist or do not list them

Comment (sawpresto) [inline docs/HOL_PartsUnlimited_HDD-User-Telemetry/HOL_PartsUnlimited_HDD-User-Telemetry.md]: Fixed!

File: docs/HOL_PartsUnlimited_HDD-User-Telemetry/HOL_PartsUnlimited_HDD-User-Telemetry.md (hunk 2 above)
Comment (sawpresto) [inline docs/HOL_PartsUnlimited_HDD-User-Telemetry/HOL_PartsUnlimited_HDD-User-Telemetry.md]: Fixed!

File: docs/HOL_PartsUnlimited_HDD-User-Telemetry/HOL_PartsUnlimited_HDD-User-Telemetry.md (hunk 1 above)
Comment (sawpresto) [inline docs/HOL_PartsUnlimited_HDD-User-Telemetry/HOL_PartsUnlimited_HDD-User-Telemetry.md]: Fixed!

**Comment by sawpresto** [comment]
Hey @dtzar , anything else that I should fix for this lab?

File: docs/HOL_PartsUnlimited_HDD-User-Telemetry/HOL_PartsUnlimited_HDD-User-Telemetry.md (hunk 2 above)
Comment (dtzar) [inline docs/HOL_PartsUnlimited_HDD-User-Telemetry/HOL_PartsUnlimited_HDD-User-Telemetry.md]: Nope - it is "telemetry" not "telemtry"

Comment (sawpresto) [inline docs/HOL_PartsUnlimited_HDD-User-Telemetry/HOL_PartsUnlimited_HDD-User-Telemetry.md]: Dangit! Missed one. :)

Comment (sawpresto) [inline docs/HOL_PartsUnlimited_HDD-User-Telemetry/HOL_PartsUnlimited_HDD-User-Telemetry.md]: Fixed it this time.

**Pull request merged**